r-travis/scripts/travis-tool.sh

```diff
@@ -27,10 +27,23 @@ PATH="${PATH}:/usr/texbin"
 R_BUILD_ARGS=${R_BUILD_ARGS-"--no-manual"}
 R_CHECK_ARGS=${R_CHECK_ARGS-"--no-manual --as-cran"}
 
-R_USE_BIOC_CMDS="source('${BIOC}');"\
+R_VERSION_TEST="paste(R.Version()$major, R.Version()$minor, sep = '.') >="\
+" package_version('3.5.0')"
```

Comment:
Maybe simpler:

``` r
getRversion() >= "3.5.0"
#> [1] TRUE
```

<sup>Created on 2018-11-28 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1.9000)</sup>

Not sure if this works for older R versions, though.

Reply:
I had a look at the history for this function and it's 10 years old so I guess it does support older R versions up to 10 years back.